sBTC SIP

```diff
@@ -0,0 +1,166 @@
+- sBTC deposits will be triggered via an API call. During the bootstrap phase sBTC deposits must be initiated via an API call to alert the signers to the presence of a deposit. After the bootstrapping phase users will have the option to alert the Signers of a deposit via a Stacks contract call, but the API will remain as a fee-less option to communicate with the Signers.
+
+These changes enable sBTC to be launched earlier and with less risk than the long term decentralized approach. The sBTC release with an open and dynamic signer set is expected to be released early 2025, which would enshrine the sBTC protocol into Stacks consensus.
+
+## Specification
+
+### Deposits (Minting sBTC)
+An API call referencing a Bitcoin transaction initiates the deposit process that can be completed within a single Bitcoin block. The API is separate from the sBTC Bootstrap Signers but will need access to the sBTC Bootstrap Signers to let them know which transactions to pick up.
+
+**API:** The API relays deposit information to the sBTC Bootstrap Signers. The API reduces the cost for deposit on rejection, reduces the scope of sBTC, and makes the sBTC system easier to service.
+
+**Bitcoin Deposit Information:** The deposit will need to pay to taproot and be spendable by a consensus threshold of the signers. The API will need to know all of the taproot spend paths and the spend paths will need to fit a specific format that cannot be clawed back in the short term in order for the Signers to consider approving it.
+
+#### Deposit Flow
+The main steps of the sBTC Deposit flow are as follows:
+1. **Deposit request:** A bitcoin holder creates a transaction on Bitcoin.
+    - The deposit transaction contains a UTXO (deposit UTXO) spendable by sBTC Signers, with an OP_DROP payload.
+    - The payload contains the recipient address of the sBTC among other relevant info for the deposit.
+    - The relevant info could contain fee suggestion or max_fee.
+2. **Proof of deposit:** The bitcoin holder submits a proof of deposit on Stacks by invoking the Signer API.
+3. **Deposit accept:**
+4. **Deposit redeem:** The sBTC Signers redeem the deposit by consuming the deposit UTXO, consolidating it into the sBTC UTXO.
+5. **Mint:** The sBTC Signers finalize the deposit acceptance by calling the mint function in .sbtc to mint the sBTC.
+
+### Withdrawals (Redeeming sBTC)
+A clarity call initiates the withdrawal process that successfully completed after 6 sortitions paying to the specified Bitcoin address. Failure cases can be resolved earlier than 6 sortitions, but some will only fail after the sBTC Bootstrap Signer attempts to create the withdrawal transaction.
```

Comment:
A few tings:

* Who initiates the clarity call to withdraw?  Is it the user, or the sBTC signer?

* You say "6 sortitions," but earlier you said "6 Bitcoin blocks."  Can you use consistent language here?

* What are the "failure cases"?  What does resolving them look like, and why is it important that it happens within 6 Bitcoin blocks?  What does a failure to resolve them look like, and who is affected by these failures (and how)?

Reply:
Described in [Clarity LLD](https://docs.google.com/document/d/1Bwi1jM_7p9OqNGT41eWCuoKnfoOrNHycqxby3XSonCA/edit?tab=t.0#heading=h.z1svj1g6pe7v). We can add all these details, linking these for reference when adding the details.

Reply:
Yup, let's add them